### DISCUSSION TITLE
lexmark toner chip reset Ti046b1

### DISCUSSION THREAD

**Discussion by rcmoh** [community/community#58630] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Does anyone have a reset code for the chips Ti046b1

**Comment by takhaabdelghani** [comment]
Hello
Does anyone have a reset code for the chips Ti046b1

**Comment by takhaabdelghani** [comment]
Hello
Does anyone have a reset code for the chips Ti046b1
![IMG20231213162549](https://github.com/community/community/assets/105175349/740ac289-a74b-4bd8-8a9f-7240fddea5b4)

**Comment by Akash1134** [comment]
Hi @rcmoh ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
